packages/backend/src/models/SpaceModel.ts

```diff
@@ -334,6 +338,7 @@ export class SpaceModel {
                 project_id: project.project_id,
                 is_private: true,
                 name,
+                created_by_user_id: userId,
```

Comment:
what happens with the spaces that we already have in the DB?

Reply:
They will be set to null. We faces this problem in the past. It is hard to find ownership dor created items